apps/src/lib/node/ledger/protocol/mod.rs

```diff
@@ -120,7 +119,24 @@ where
                 ibc_event,
             })
         }
-        _ => {
+        TxType::Protocol(ProtocolTx {
+            tx: ProtocolTxType::EthereumEventVoteExtensions,
+            ..
+        }) => {
+            tracing::debug!(
+                "EthereumEventVoteExtensions protocol transaction received"
+            );
+            let gas_used = block_gas_meter
+                .finalize_transaction()
+                .map_err(Error::GasError)?;
+            Ok(TxResult {
+                gas_used,
+                ..Default::default()
+            })
+        }
+        TxType::Protocol(_)
+        | TxType::Wrapper(_)
+        | TxType::Decrypted(DecryptedTx::Undecryptable(_)) => {
```

Comment:
I expanded the `_` here to make explicit what we are executing this match arm for, it looks like a sort of noop but I'm not sure - the previous docstring mentioned the only tx types that should be input to this fn are `Decrypted` and `Wrapper`, so should we be erroring here instead?